packages/ipfs-http-client/test/node/agent.js

```diff
@@ -0,0 +1,119 @@
+/* eslint-env mocha */
+'use strict'
+
+const { isNode } = require('ipfs-utils/src/env')
+const { expect } = require('aegir/utils/chai')
+const ipfsClient = require('../../src')
+const delay = require('delay')
+
+const PORT = 29832
+
+function startServer (handler) {
+  return new Promise((resolve) => {
+    // spin up a test http server to inspect the requests made by the library
+    const server = require('http').createServer((req, res) => {
+      req.on('data', () => {})
+      req.on('end', async () => {
+        const out = await handler(req)
+
+        res.writeHead(200)
+        res.write(JSON.stringify(out))
+        res.end()
+      })
+    })
+
+    server.listen(PORT, () => {
+      resolve({
+        close: () => server.close()
+      })
+    })
+  })
+}
+
+describe('agent', function () {
+  // do not test in browser
+  if (!isNode) {
+    return
+  }
```

Comment:
this isnt needed, only the .spec.js files run in all envs

Reply:
Have removed, I think it just came from the headers test.